### ISSUE TITLE
Failed to analyze Apache Xerces

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ankit4u3/jtracert#9]
```
What steps will reproduce the problem?
1. Create a simple application which utilizes Xerces as described in Xerces
tutorial: http://www.totheriver.com/learn/xml/xmltutorial.html#6.1.1
2. Execute it together with jTracert

What is the expected output? What do you see instead?

jTracert should analyze the application and build the sequence diagrams.
Instead we get a null pointer exception as shown below:

 java.lang.NullPointerException
    at
com.google.code.jtracert.traceBuilder.impl.MethodCallTraceBuilderImpl.leave(Meth
odCallTraceBuilderImpl.java:323)
    at com.google.code.jtracert.samples.Xerces.main(Unknown Source)

Please use labels and text to provide additional information.

GNU/Linux
java version "1.6.0_07"
Java(TM) SE Runtime Environment (build 1.6.0_07-b06)
Java HotSpot(TM) 64-Bit Server VM (build 10.0-b23, mixed mode)


```

Original issue reported on code.google.com by `Dmitry.Bedrin` on 9 Feb 2009 at 3:05

**Comment by GoogleCodeExporter** [comment]
```
Sample application 'xerces' is now added to SVN
```

Original comment by `Dmitry.Bedrin` on 9 Feb 2009 at 3:43

**Comment by GoogleCodeExporter** [comment]
```
The problem is actually not concerned with Xerces.

jTracert incorrectly analyzes constructors - when one constructor calls the 
other
one, or parent constructor. We cannot insert custom code before parent 
constructor
call, so this can be a problem.

Priority raised to Critical.
```

Original comment by `Dmitry.Bedrin` on 10 Feb 2009 at 9:26
- Added labels: **Component-Logic**, **Priority-Critical**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
I've created a separate branch for fixing this defect, since it may require
significant changes and may take a lot of time.
http://code.google.com/p/jtracert/source/browse/#svn/branches/issue9
```

Original comment by `Dmitry.Bedrin` on 10 Feb 2009 at 10:58

**Comment by GoogleCodeExporter** [comment]
```
Fixed in svn revision r126 ( issue9 branch )
```

Original comment by `Dmitry.Bedrin` on 17 Feb 2009 at 10:23
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in jTracert version 0.0.5
```

Original comment by `Dmitry.Bedrin` on 4 Mar 2009 at 1:36
- Changed state: **Verified**
